app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+class Post < ActiveRecord::Base
+  has_many :comments, dependent: :destroy
+  belongs_to :user
+
+  has_attached_file :image, :styles => { :medium => "450x450>", :thumb => "100x100>" },
+                    :default_url => "missing.png"
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)